### DISCUSSION TITLE
Geeting a white screen

### DISCUSSION THREAD

**Discussion by sushant4612** [ruxailab/RUXAILAB#350] [Q&A] [answered]
I have completed the setup, and it was running fine until yesterday. However, today it's showing only a white screen after running `npm run serve`.

I haven't made any changes. I even tried cloning the repo again, but I'm still encountering the same issue.

<img width="1466" alt="Screenshot 2024-03-06 at 6 57 12 PM" src="https://github.com/ruxailab/RUXAILAB/assets/109458485/f3e8d1ec-a304-4746-96d6-cd86cc2bd7ca">

**Comment by uchiha-vivek** [comment]
Yes this issue exist

**Reply by sushant4612** [reply]
you know any solution for this

**Reply by uchiha-vivek** [reply]
woring on it

**Reply by uchiha-vivek** [reply]
Once try to check the firebase configuration

**Comment by jvJUCA** [comment, marked as answer]
Dear @sushant4612,

To ensure you have all the necessary instructions to address this issue, I would recommend you to review this discussion: [discussion of this run issues](https://github.com/ruxailab/RUXAILAB/discussions/309), and also watch our tutorial video on how to run the project.

[Link to the tutorial video - "Tutorial - Running RUXAILAB with Firebase Emulators 🖥️"](https://youtu.be/dAf4LRxITCc)

Best regards,

@jvJUCA

**Reply by sushant4612** [reply]
Thanks for the response @jvJUCA it's working now

**Reply by sushant4612** [reply]
Hey @jvJUCA , I am looking to contribute to an issue, but every issue is assigned to someone. Can I still try to solve it and raise a PR for it?

**Comment by jvJUCA** [comment]
> Hey @jvJUCA , I am looking to contribute to an issue, but every issue is assigned to someone. Can I still try to solve it and raise a PR for it?

Dear @sushant4612 ,

Currently we have a PR that is to message in alerts be converted to toastify, but the contributor runned npm run lint and i cant review, if you can do this issue without linting, will be soo helpful!!!!

Best regards,

@jvJUCA

**Reply by sushant4612** [reply]
Yes I will try that

**Comment by KarinePistili** [comment]
Everyone, I'll be closing this discussion as we are concentrating this question on another one #309 .
